src/components/nav-bar/nav-bar.js

```diff
@@ -66,10 +67,12 @@ const NavBar = () => {
     </IconButton>
   );
 
+  const urlPage = window.location.pathname;
+
   return (
     <AppBar className={classes.appBar}>
       <Toolbar>
-        {menuToggle}
+        {urlPage === '/login' ? null : menuToggle}
```

Comment:
Please, don't use magic string
You can find appropriate const in src/configs/routes.js